Room/app/src/main/java/com/example/room/di/DataBaseModule.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.room.di
+
+import android.content.Context
+import androidx.room.Room
+import com.example.room.data.local.db.FriendDAO
+import com.example.room.data.local.db.FriendDatabase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DataBaseModule {
+
+    @Singleton
+    @Provides
+    fun provideFriendDatabase(@ApplicationContext context: Context): FriendDatabase {
+        return Room.databaseBuilder(context, FriendDatabase::class.java, "friend_data_database")
+            .build()
+    }
```

Comment:
```suggestion
    fun provideFriendDatabase(@ApplicationContext context: Context) = Room.databaseBuilder(context, FriendDatabase::class.java, "friend_data_database")
            .build()
```

굳이 바꾸시지 않아도 됩니다! 하지만 이렇게도 표현 가능하다는 거 이미 알고계시는 거 다 압니다 ^_____^

Reply:
아 주의하겠습니다!